http-client/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
-name = "wascc-httpclient"
-version = "0.1.0"
-authors = ["Dan Norris <protochron@users.noreply.github.com>"]
+name = "wasmcloud-httpclient"
+version = "0.2.0"
+authors = ["wasmCloud Team"]
 edition = "2018"
-repository = "https://github.com/wascc/http-client-provider"
-description = "HTTP client capability provider for the waSCC wasm host runtime"
+repository = "https://github.com/wasmcloud/http-client-provider"
```

Comment:
this URL is incorrect, it should be https://github.com/wasmcloud/capability-providers